webserver/views/api.py

```diff
@@ -65,10 +78,10 @@ def submit_low_level(mbid):
     try:
         data = json.loads(raw_data.decode("utf-8"))
     except ValueError as e:
-        raise BadRequest("Cannot parse JSON document: %s" % e)
+        raise webserver.exceptions.InvalidAPIUsage("Cannot parse JSON document: %s" % e, status_code=BadRequest.code)
 
     try:
         submit_low_level_data(mbid, data)
     except BadDataException as e:
-        raise BadRequest(e)
-    return ""
+        raise webserver.exceptions.InvalidAPIUsage("%s" % e, status_code=BadRequest.code)
+    return jsonify({"status": "ok"})
```

Comment:
though, the exception handler returns a key `message`. Perhaps we should be consistent.